docs/reference/filters.md

```diff
@@ -1084,6 +1084,98 @@ forwardToken("X-Tokeninfo-Forward")
 forwardToken("X-Tokeninfo-Forward", "access_token")
 ```
 
+## oauthGrant
+
+Enables authentication and authorization with an OAuth2 authorization code grant flow.
```

Comment:
should we add a reference to the RFC https://tools.ietf.org/html/rfc6749#section-1.3.1

Reply:
Done